content/departments/product-engineering/product/design/workshops/async-design.md

```diff
@@ -0,0 +1,63 @@
+
+The activities you do on a given day are flexible with this approach, so you can do problem validation, solution validation, brainstorming, and more; the important part is making it a multi-day activity with offsetting days for content creation and evaluation/voting, with a date boundary in between.
+
+Benefits:
+
+- Supports async work
+- More flexibility when people are in lots of time zones
+- Can support quite large groups
+- Self-documenting
+- Allows time for reflection and "shower thoughts" to be included
+
+Drawbacks:
+
+- Plays out slower than if everyone was in a live session together
+- Not as engaging for people who are inspired by real-time communication
+- Requires lots of written communication
+
+## Scheduling considerations
+
+Because this is an activity that takes a little bit of attention each day over a few days or a week, it's good to avoid scheduling at the same time as the release is wrapping up, or another heavyweight organizational activity like planning is going on.
+
+## Offsetting days
+
+The first thing you should do when planning the workshop is identify the list of activities you want to complete. Then, order them across multiple days with one day doing the activity, and the next day summarizing and voting. This rhythm of having a day for producing information and a day for summarizing it/voting on it keeps the team making forward progress, and supports people participating in all time zones. It also is highly tolerant of people joining late or dropping out early.
+
+See [this example of a complete agenda](#example-agenda) for how you might structure something like this in detail.
+
+## Deliverables
+
+On the non-offset days, it's important to structure your deliverables so that they can be shared easily. Simple is best - letting people share off the cuff thoughts in the Slack channel has the lowest barrier to entry, and will encourage the most participation. Beyond that, written tends to be ideal, but videos are also good. So, if you're planning on doing some problem validation, you might have everyone record a short video of problems they are aware of with the product today. These are easily watchable by everyone, and it's also easy to organize them into groupings the next day and vote on them.
+
+In cases where you do want a very specifically formatted output (for example, solution specifications) you should include a template that you want the team to use. Just be mindful that this increases the barrier for entry and may reduce the number of submissions, although sometimes it is necessary.
+
+## Coordination
+
+You can coordinate the activity via a Slack channel. It is recommended to use a new, temporary channel for this purpose to avoid confusion from mixing messages. It can be helpful to have multiple facilitators spread across time zones to ensure someone is always around to answer any questions that come up.
```

Comment:
```suggestion
You can coordinate the activity via a Slack channel. GitHub Discussions are also a good tool channel for discussion as it makes ideas public, and is a better support for having long-running conversations and voting on topics than Slack. It is recommended to use a new, temporary channel for this purpose to avoid confusion from mixing messages. It can be helpful to have multiple facilitators spread across time zones to ensure someone is always around to answer any questions that come up.
```

Reply:
Thanks!